src/libs/elektra/internal.c

```diff
@@ -715,6 +715,21 @@ size_t elektraUnescapeKeyName (const char * source, char * dest)
 
 	ELEKTRA_ASSERT (sp != NULL && dp != NULL, "Got null pointer sp: %p dp: %p", (void *) sp, (void *) dp);
 
+	// if there is nothing to unescape, just make a copy and replace / with \0
+	if (strpbrk (sp, "\\%") == NULL)
```

Comment:
Is this a performance improvement? Why not put it into a separate function?

Reply:
Yes, this should improve performance. The function was so short already, it didn't seem necessary to split it up.